### PR TITLE
More functional classes and data-test selectors

### PR DESCRIPTION
**Pull request by nummi**
Moved a lot of styles from `app/styles` to functional classes.

(also moved a bunch of test selectors to `data-test`)

### CODE REVIEW

**Comment by rwwagner90** [comment]
Will review this sometime this week. Apologies for the delay!

File: app/components/object-inspector.hbs [hunk 1]
```diff
@@ -2,43 +2,52 @@
...
       <button
-        class="toolbar__icon-button {{if this.isNested "enabled" "disabled"}} js-object-inspector-back"
+        class="toolbar__icon-button {{if this.isNested "enabled" "disabled"}}"
         {{on "click" this.popStack}}
+        data-test-object-inspector-back
       >
         {{svg-jar "arrow-back" width="21px" height="21px"}}
       </button>
 
-      <Ui::ToolbarDivider />
```
Comment (rwwagner90) [inline app/components/object-inspector.hbs:11]: We no longer need this divider?

Comment (nummi) [reply app/components/object-inspector.hbs:11]: To me it seems like extra noise.

image

image

image

image

**Comment by rwwagner90** [comment]
@chancancode any ideas why GitHub actions refuses to run here?

**Comment by rwwagner90** [comment]
@nummi can you try rebasing or pushing just a new space added or something? Trying to kick GitHub actions into actually running.

**Comment by chancancode** [comment]
> Volta error: Could not unpack Node v10.19.0

This means it couldn't download the version of node needed for the build. https://github.com/nodejs/nodejs.org/issues/3075

**Comment by rwwagner90** [comment]
@nummi looks like we have a conflict here. Sorry about that! Do you mind resolving? I think we can merge after that.

**Pull request merged**